IFC4x3/Properties/i/IsSpecialTransportRoute_1UpphLsIf8AQz1eoTOneG9/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsSpecialTransportRoute_1UpphLsIf8AQz1eoTOneG9" Name="IsSpecialTransportRoute" UniqueId="5ecf3ad5-d92a-4829-af41-a32758c68409" PrimaryDataType="IfcBoolean" />
```

Comment:
According to the description propose renaming to `IsExceptionalTransportRoute`.

Reply:
Agree, updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/5eb5d2a2bb92b629c57058b376a6ae07d988a357